libs/shared/constants/src/lib/projectBasePath.ts

```diff
@@ -0,0 +1,7 @@
+export enum ProjectBasePath {
+  ServicePortal = '/minarsidur',
+  ConsultationPortal = '/samradsgatt',
+  ApplicationSystem = '/umsoknir',
+  AirDiscountScheme = '/loftbru',
```

Comment:
AirDiscountScheme has been moved so `/loftbru` is actually a page on the web and the application link is now `/minarsidur/loftbru`. So we should drop this so we don't render `<a>` tag for a internal link.